scripts/compliance/generate_sbom.sh

```
@@ -11,28 +11,56 @@ echo "Generating SBOMs for Summit..."
-      syft dir:apps/web -o cyclonedx-json > "$OUTPUT_DIR/web-sbom.json"
+    add_sbom "web" "apps/web"
   fi
 
   # PSC Runner SBOM (Rust)
   if [ -f rust/psc-runner/Cargo.toml ]; then
-    echo "Generating PSC Runner SBOM..."
-    syft dir:rust/psc-runner -o cyclonedx-json > "$OUTPUT_DIR/psc-runner-sbom.json"
+    add_sbom "psc-runner" "rust/psc-runner"
+  fi
+
+  for root in apps packages services workers; do
```

Shellcheck (0.11.0)

[error] Couldn't parse this for loop. Fix to allow more checks.

(SC1073)